### PR TITLE
 Revert to only check local BTC node's port

### PR DESCRIPTION
**Pull request by dmos62**
Revert to only check local BTC node's port

Testing showed that the new mechanic for checking a local BTC node's
configuration is unstable. This commit reverts to just checking if the
relevant port is open. The recent refactoring and centralization of
logic is still in place.

Supersedes PR #4062.

### CODE REVIEW

**Review by ripcurlx — DISMISSED** [review]
ACK

Works for me with and without a Bitcoin Core running locally

**Comment by ripcurlx** [comment]
Waiting for another ACK by @wiz until I merge and cherry-pick it into the v1.2.9 release branch

**Review by wiz — DISMISSED** [review]
ACK
tested OK on 4 node linux regtest setup :+1:

File: core/src/main/resources/i18n/displayStrings.properties
Comment (sqrrm) [inline core/src/main/resources/i18n/displayStrings.properties]: Should this message remain without the proper check?

Comment (dmos62) [reply core/src/main/resources/i18n/displayStrings.properties]: Didn't notice that. Should be changed. Have to go out for an hour; will change it when I come back.

Comment (sqrrm) [inline core/src/main/resources/i18n/displayStrings.properties]:
```suggestion
popup.bitcoinLocalhostNode.additionalRequirements=\n\nFor a well configured node, the requirements are for the node to have pruning disabled and bloom filters enabled.
```

Comment (ripcurlx) [reply core/src/main/resources/i18n/displayStrings.properties]: Good catch - this should be in that case
```suggestion
popup.bitcoinLocalhostNode.additionalRequirements=\n\nStarting with Bitcoin Core v0.19 you have to manually enable bloom filters by setting peerbloomfilters=1 in your bitcoin.conf configuration file.
```

File: core/src/main/resources/i18n/displayStrings.properties
Comment (ripcurlx) [inline core/src/main/resources/i18n/displayStrings.properties]: This should be:
```suggestion
popup.bitcoinLocalhostNode.msg=Bisq detected a locally running Bitcoin Core node (at localhost).\n\
  Please make sure that this node is fully synced before you start Bisq and that it is not running in pruned mode.
```

**Comment by ripcurlx** [comment]
@dmos62 I already committed the change suggestions

**Review by ripcurlx — APPROVED** [review]
ACK

Committed the suggested text changes

**Pull request merged**
